sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -932,14 +932,14 @@ object RapidsConf {
       .internal()
       .doc("Default cost of transitioning from GPU to CPU")
       .doubleConf
-      .createWithDefault(0.15)
+      .createWithDefault(0.1)
```

Comment:
The cost model is very arbitrary but the default values for GPU are that operators are 0.8 and transitions are now 0.1 (there are not time units yet but relative costs compared to CPU). So for an operator with two children that both have transitions from CPU, the GPU cost now would be 0.8 + 0.1 + 0.1 = 1.0 so same as CPU whereas before it would have been 0.8 + 0.15 + 0.15 = 1.1 which is > CPU cost of 1.0 and we were trying to push all joins back onto CPU by default.